code/modules/projectiles/gun.dm

```diff
@@ -195,7 +199,8 @@
 			if(G == src || G.weapon_weight >= WEAPON_MEDIUM)
 				continue
 			else if(G.can_trigger_gun(user))
-				bonus_spread += 24 * G.weapon_weight
+				if(!HAS_TRAIT(user, TRAIT_BADASS))
+					bonus_spread += 24 * G.weapon_weight
```

Comment:
This is just my opinion, but this will be VERY powerful, 2 stechs and this will cost 50 tc, while the same damaging .357 revolver will be 15 tc more expensive. This should probably only REDUCE the spread, not remove it entirely

Reply:
2 steches being dual-wielded are easier to disarm, easier to dodge, harder to reload, and consume much more ammo and inventory space. You also can't quick-draw two stechkins at once, meaning you have more time to react to seeing the inhands + cigar than you would with a .357